reposilite-plugins/javadoc-plugin/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.reposilite.plugins"
+
+plugins {
+    kotlin("jvm")
+}
+
+application {
+    mainClass.set("com.reposilite.plugin.javadoc.JavadocPluginKt")
+}
+
+dependencies {
+    // compileOnly("org.panda-lang:reposilite:version") for external plugins
+
+    implementation("net.dzikoysk:exposed-upsert:1.0.3")
```

Comment:
Should be `compileOnly`, because `implementation` + `shadow` plugin will bundle dependency in plugin jar and we don't want it to avoid conflict between classes of the same dependency at runtime